src/webui/api/torrentscontroller.cpp

```diff
@@ -1097,10 +1097,9 @@ void TorrentsController::addTagsAction()
     const QStringList tags {params()["tags"].split(',', QString::SkipEmptyParts)};
 
     for (const QString &tag : tags) {
-        const QString tagTrimmed {tag.trimmed()};
```

Comment:
I checked again in GUI, tag name cannot be mere whitespace.

Reply:
@Chocobo1
Fixed.